### PR TITLE
fix: Pull databaseIds for TabbedSqlEditor Component

### LINKED ISSUE #11914: SQL Lab - Database Not Selected by Default
When going into SQL Lab for the first time, a database should be selected by default. Currently, the dropdown is blank and there's not clear feedback to the user that they need to select a database if they try to run a query. 

### Expected results
The first database in the list should be populated by default when the user visits SQL Lab 

### Actual results
No database is populated 

#### Screenshots

image

#### How to reproduce the bug

1. Visit SQL Lab for the first time 
2. In your left panel, see if anything is selected in the database dropdown

Alternatively: 
1. Close all your query tabs in SQL Lab
2. Visit SQL Lab again by clicking it in the header or refreshing your page 
2. In your left panel, see if anything is selected in the database dropdown

### Environment

- superset version: `master`

### Checklist

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### PR DESCRIPTION
**Pull request by hughhhh**
### SUMMARY

DatabaseSelector is missing a default value whenever the user refreshes on an empty tab page in sqllab. Now we've added a request to get all the available `databaseIds` in the constructor.

fixes https://github.com/apache/incubator-superset/issues/11914

https://www.loom.com/share/3944f97db5f442858fd20635a1030c83

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx [hunk 1]
```diff
@@ -32,6 +32,22 @@ import * as Actions from '../actions/sqlLab';
...
+
+
+const getDatabasesIds = async () => {
+  const queryParams = rison.encode({});
+  const response = await makeApi({
+    method: 'GET',
+    endpoint: '/api/v1/database',
+  })(`q=${queryParams}`);
+
+  console.log(response.ids);
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:47]: dd

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx [hunk 2]
```diff
@@ -240,11 +256,10 @@ class TabbedSqlEditors extends React.PureComponent {
   }
 
   newQueryEditor() {
+    console.log(this.state.databaseIds);
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx]: dd

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx [hunk 3]
```diff
@@ -32,6 +32,22 @@ import * as Actions from '../actions/sqlLab';
...
+import { makeApi } from '@superset-ui/core';
+import rison from 'rison';
+
+
+const getDatabasesIds = async () => {
+  const queryParams = rison.encode({});
+  const response = await makeApi({
+    method: 'GET',
+    endpoint: '/api/v1/database',
+  })(`q=${queryParams}`);
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:41]: Do you need the params here?

**Review by eschutho — APPROVED** [review]
looks great

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx (hunk 3 above)
Comment (hughhhh) [reply superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:41]: nope lol

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx [hunk 4]
```diff
@@ -32,6 +32,22 @@ import * as Actions from '../actions/sqlLab';
...
+
+  console.log(response.ids);
+  return response;
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:41]: Better to return `response.ids` here, to be consistent with the function name.

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx [hunk 5]
```diff
@@ -240,11 +256,10 @@ class TabbedSqlEditors extends React.PureComponent {
   }
 
   newQueryEditor() {
+    console.log(this.state.databaseIds);
     queryCount += 1;
     const activeQueryEditor = this.activeQueryEditor();
-    const firstDbId = Math.min(
-      ...Object.values(this.props.databases).map(database => database.id),
-    );
+    const firstDbId = Math.min(this.state.databaseIds);
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:254]: This won't work, right? Right now `getDatabasesIds` returns a list of databases.

**Review by betodealmeida — APPROVED** [review]
Looks good now!

File: superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx (hunk 5 above)
Comment (hughhhh) [reply superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx:254]: databaseIds is an array<numbers>

**Comment by mistercrunch** [comment]
Closing in favor of https://github.com/apache/incubator-superset/pull/11922 (?)

**Pull request closed without merging**